Classes/Providers/Base/FormatsHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FormatsHelper.swift
+//  Pods
+//
+//  Created by Rivka Peleg on 05/03/2017.
+//
+//
+
+import Foundation
+
+public class FormatsHelper {
```

Comment:
Why public? should be available for objc apps also? if yes please add @objc to all public declarations